pkg/crc/machine/bundle/repository.go

```diff
@@ -89,11 +81,23 @@ func (bundle *CrcBundleInfo) installSymlinkOrCopy(ocBinDir string) error {
 	return os.Symlink(ocInBundle, ocInBinDir)
 }
 
-func Extract(sourcepath string) (*CrcBundleInfo, error) {
-	_, err := extract.Uncompress(sourcepath, constants.MachineCacheDir, true)
-	if err != nil {
+func (r *Repository) Extract(path string) error {
+	_, err := extract.Uncompress(path, r.CacheDir, true)
+	return err
+}
+
+var defaultRepo = &Repository{
+	CacheDir: constants.MachineCacheDir,
+	OcBinDir: constants.CrcOcBinDir,
+}
+
```

Comment:
The introduction of `defaultRepo` could have been squashed in the previous commit rather than here.

Reply:
It was only once in the previous commit, the commit was already large, no need for it ;)